cli/cmd/plugin/unmanaged-cluster/cluster/minikube.go

```diff
@@ -0,0 +1,310 @@
+// Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cluster
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/vmware-tanzu/community-edition/cli/cmd/plugin/unmanaged-cluster/config"
+
+	"k8s.io/klog/v2"
```

Comment:
If we did https://github.com/vmware-tanzu/community-edition/pull/3674/files#r842201869, I believe we'd also get rid of our **new** `klog` dep, which would be great :+1:.

Reply:
True true - will remove and serialize the json output